handles.go

```diff
@@ -0,0 +1,198 @@
+// Copyright 2021 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package frankenphp
+
+import (
+	"sync"
+	"sync/atomic"
+)
+
+// Handle provides a way to pass values that contain Go pointers
```

Comment:
Maybe you could add a link to your CL and your blog post, or quickly explain the changes that have been made compared to what is in the standard library.

I would also drop all the documentation (it will be painful to maintain, a link to the official website will be enough.